### DISCUSSION TITLE
auth issue

### DISCUSSION THREAD

**Discussion by mahdi2k1** [adonisjs/core#2586] [Help] [answered]
{
    "code": "MODULE_NOT_FOUND",
    "requireStack": [
        "E:\\js\\adonis js v5\\hello-world\\node_modules\\@adonisjs\\hash\\build\\src\\Drivers\\Argon.js",
        "E:\\js\\adonis js v5\\hello-world\\node_modules\\@adonisjs\\hash\\build\\src\\Hash\\index.js",
        "E:\\js\\adonis js v5\\hello-world\\node_modules\\@adonisjs\\hash\\build\\standalone.js",
        "E:\\js\\adonis js v5\\hello-world\\node_modules\\@adonisjs\\core\\build\\standalone.js",
        "E:\\js\\adonis js v5\\hello-world\\server.ts"
    ]
}

**Comment by RomainLanz** [comment, marked as answer]
Hey @mahdi2k1! 👋 

Ensure to have installed the hash module you are using.
▶️ https://docs.adonisjs.com/guides/security/hashing
